website/client/js/controllers/memberModalCtrl.js

```diff
@@ -19,8 +19,10 @@ habitrpg
           $scope.profile = member;
         }
       });
-      
-      $scope.formatAnimal = Costume.formatAnimal;
+
+      for (var method in Costume) {
```

Comment:
Maybe it's just me, but I think the methods should be stated explicitly.  That way, it's easy to tell what methods are in scope without digging around in other files.